src/applications/disability-benefits/all-claims/components/TraumaticEventTypesPage.jsx

```diff
@@ -0,0 +1,294 @@
+        );
+      });
+    }
+
+    return false;
+  };
+
+  const deleteMSTEvidence = () => {
+    const deepClone = cloneDeep(data);
+
+    if (deepClone.optionIndicator) {
+      deepClone.optionIndicator = undefined;
+    }
+
+    deepClone.events?.forEach(event => {
+      Object.assign(event.militaryReports, {
+        // TODO: ideally we should use the MILITARY_REPORT_TYPES constant here
+        restricted: false,
+        unrestricted: false,
+        pre2005: false,
+      });
+    });
+
+    setFormData(deepClone);
+  };
+
+  // If the user is opts to deselect MST and they are asked in the modal to delete their MST data,
+  // they can close the modal or click the Cancel button. Doing so should revert the MST checkbox back to checked
+  const revertMSTEventTypeDeselection = () => {
+    const deepClone = cloneDeep(data);
+
+    const traumaticEventsSelections = deepClone.eventTypes;
+    // Re-selct MST box
+    traumaticEventsSelections.mst = true;
+    deepClone.eventTypes = traumaticEventsSelections;
+
+    setFormData(deepClone);
+  };
+
+  const shouldShowDeleteMSTEvidenceModal = () => {
+    return (
+      data.eventTypes?.mst === false &&
+      (data.optionIndicator || claimedIncidentReportsForEvents())
+    );
+  };
+
+  const handlers = {
+    onSelectionChange: event => {
+      const { target } = event;
+      const selection = event.target?.getAttribute('value');
+
+      const formData = {
+        ...data,
+        eventTypes: {
+          ...data.eventTypes,
+          [selection]: target.checked,
+        },
+      };
+      setFormData(formData);
+    },
+    onSubmit: event => {
+      event.preventDefault();
+
+      if (shouldShowDeleteMSTEvidenceModal()) {
+        setShowDeleteMSTEvidenceModal(true);
+      } else {
+        goForward(data);
+      }
+    },
+    onCloseModal: () => {
+      revertMSTEventTypeDeselection();
+      setShowDeleteMSTEvidenceModal(false);
+    },
+    onConfirmDeleteMSTEvidence: () => {
+      deleteMSTEvidence();
+      setShowDeleteMSTEvidenceModal(false);
+      setShowDeletedEvidenceConfirmation(true);
+    },
+    onCancelDeleteBehavioralAnswers: () => {
+      handlers.onCloseModal();
+    },
+    onCloseDeletedEvidenceAlert: () => {
+      setShowDeletedEvidenceConfirmation(false);
+    },
+    onClickConfirmationLink: () => {
+      goForward(data);
+    },
+    onUpdatePage: event => {
+      event.preventDefault();
+
+      if (shouldShowDeleteMSTEvidenceModal()) {
+        setShowDeleteMSTEvidenceModal(true);
+      } else {
+        updatePage(event);
+      }
+    },
+  };
+
+  return (
+    <div className="vads-u-margin-y--2">
+      <div className="vads-u-margin-bottom--1">
+        <VaAlert
+          ref={deletedEvidenceConfirmationRef}
+          closeBtnAriaLabel="Deleted MST evidence confirmation"
+          closeable
+          onCloseEvent={handlers.onCloseDeletedEvidenceAlert}
+          fullWidth="false"
+          slim
+          status="warning"
```

Comment:
All the other alerts are green, with the "success" status. Is there a reason this one is different? 

```suggestion
          status="success"
```

Reply:
Confirmed with Ruben, the "warning" status is correct for this alert because there are other instructions for the user to note.